clients/Admin/config-asset-loader.ts

```diff
@@ -0,0 +1,27 @@
+import { Injectable } from '@angular/core';
```

Comment:
Do we need this file in reference architecture? I believe this file is for loading properties when we pre-provision build labs for workshop.